tests/foreman/cli/test_activationkey.py

```diff
@@ -49,1578 +49,1577 @@
+
+
+@pytest.mark.tier2
+@pytest.mark.parametrize('host_col_name', **parametrized(valid_data_list()))
+def test_positive_update_host_collection(module_org, host_col_name):
+    """Test that host collections can be associated to Activation
+    Keys
+
+    :id: 2114132a-fede-4791-98e7-a463ad79f398
+
+    :BZ: 1110476
+
+    :expectedresults: Host collections are successfully associated to
+        Activation key
+
+    :CaseLevel: Integration
+
+    :parametrized: yes
+    """
+    activation_key = make_activation_key({'organization-id': module_org.id})
+    new_host_col_name = make_host_collection(
+        {'name': host_col_name, 'organization-id': module_org.id}
+    )['name']
+    # Assert that name matches data passed
+    assert new_host_col_name == host_col_name
+    ActivationKey.add_host_collection(
+        {
+            'host-collection': new_host_col_name,
+            'name': activation_key['name'],
+            'organization-id': module_org.id,
+        }
+    )
+    activation_key = ActivationKey.info({'id': activation_key['id']})
+    assert activation_key['host-collections'][0]['name'] == host_col_name
+
+
+@pytest.mark.run_in_one_thread
+@pytest.mark.tier2
+def test_positive_update_host_collection_with_default_org(module_org):
+    """Test that host collection can be associated to Activation
+    Keys with specified default organization setting in config
+
+    :id: 01e830e9-91fd-4e45-9aaf-862e1fe134df
+
+    :expectedresults: Host collection is successfully associated to
+        Activation key
+
+    :BZ: 1364876
+    """
+    Defaults.add({'param-name': 'organization_id', 'param-value': module_org.id})
+    try:
+        activation_key = make_activation_key({'organization-id': module_org.id})
+        host_col = make_host_collection()
+        ActivationKey.add_host_collection(
+            {'host-collection': host_col['name'], 'name': activation_key['name']}
         )
-        env = make_lifecycle_environment({'organization-id': self.org['id']})
-        new_cv = make_content_view({'organization-id': self.org['id']})
-        ContentView.publish({'id': new_cv['id']})
-        cvv = ContentView.info({'id': new_cv['id']})['versions'][0]
-        ContentView.version_promote({'id': cvv['id'], 'to-lifecycle-environment-id': env['id']})
-        ActivationKey.update(
+        activation_key = ActivationKey.info({'id': activation_key['id']})
+        assert activation_key['host-collections'][0]['name'] == host_col['name']
+    finally:
+        Defaults.delete({'param-name': 'organization_id'})
+
+
+@pytest.mark.run_in_one_thread
+@skip_if_not_set('fake_manifest')
+@pytest.mark.tier3
+def test_positive_add_redhat_product(module_org):
+    """Test that RH product can be associated to Activation Keys
+
+    :id: 7b15de8e-edde-41aa-937b-ad6aa529891a
+
+    :expectedresults: RH products are successfully associated to Activation
+        key
+
+    :CaseLevel: System
+    """
+    org = make_org()
```

Comment:
Is this necessary when you can call `module_org`?

Reply:
certain tests fail when they run back to back using the same fixture, I had to create a separate org for these tests